nixos/modules/services/misc/gitlab.nix

```diff
@@ -21,14 +21,15 @@ let
       username: ${cfg.databaseUsername}
       encoding: utf8
   '';
+
   gitlabShellYml = ''
-    user: gitlab
-    gitlab_url: "http://${cfg.host}:${toString cfg.port}/"
+    user: ${cfg.user}
+    gitlab_url: "http://localhost:8080/"
```

Comment:
Yup, if you're using this nixos module this is constant because the listening socket is hardcoded in the unicorn config. The problem here was that gitlab-shell somehow wouldn't work with https (I changed the scheme of course :panda_face:).

But thinking about it, we should probably make this configurable somehow because 8080 is a very heavily used default port that could be already in use by some other service. I will check if gitlab shell supports unix sockets, then we can get rid of the tcp socket altogether. I will look into it and make it at least configurable.